cobra2snooty.go

```diff
@@ -34,13 +34,13 @@ const (
 	defaultExtension = ".txt"
 )
 
-// GenSnootyTree generates the docs for the full tree of commands.
-func GenSnootyTree(cmd *cobra.Command, dir string) error {
```

Comment:
the usage of this is `cobra2snooty.GenSnootyTree` in the spirit of go don't stutter, `cobra2snooty.GenTreeDocs` is a better name